pkg/apis/integreatly/v1alpha1/gitea_types.go

```diff
@@ -18,7 +18,8 @@ type GiteaSpec struct {
 type GiteaStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	Phase int `json:"phase"`
+	Phase        int    `json:"phase"`
+	GiteaVersion string `json:"giteaVersion"`
```

Comment:
ok to just name this Version to match the 3scale one

Reply:
done